include/dem/pw_linear_force.h

```diff
@@ -60,9 +60,10 @@ class PWLinearForce : public PWContactForce<dim>
                                                    boundary_translational_velocity,
     const std::unordered_map<unsigned int, double> boundary_rotational_speed,
     const std::unordered_map<unsigned int, Tensor<1, dim>>
-                                    boundary_rotational_vector,
-    const double                    triangulation_radius,
-    const DEMSolverParameters<dim> &dem_parameters);
+                                          boundary_rotational_vector,
+    const double                          triangulation_radius,
+    const DEMSolverParameters<dim> &      dem_parameters,
+    const std::vector<types::boundary_id> boundary_index = {0});
```

Comment:
Why does the boundary_index need to have a default value. Shouldnt the default vector be  an empty vector? It's not clear to me.

Reply:
I wanted to have an empty vector as default but didn't find, will retry for next commit ^^

Reply:
Make it an empty vector, yeah.